.env.example

```
@@ -26,6 +26,9 @@ OMNISEARCH_API_KEY=
 # LLM calls it has no time to finish.
 JASA_SEARCH_TIMEOUT_MS=50000
 JASA_SEARCH_FANOUT_TIMEOUT_MS=25000
+# MCP returns this many ranked rows plus eligible tail rescues. Grounding has
+# its own lower cap below, so increasing this does not buy more page fetches.
+JASA_SEARCH_MAX_RESULTS=50
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JASA_SEARCH_MAX_RESULTS key should go before the JASA_SEARCH_TIMEOUT_MS key

(UnorderedKey)